WombatProject/Services/BookRepository.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WombatLibrarianApi.Models;
+
+namespace WombatLibrarianApi.Services
+{
+    public class BookRepository : IBookRepository
+    {
+        protected WombatBooksContext _context { get; }
```

Comment:
Can't _context be private?

Reply:
I thought about that option but I figured protected is great if there would be a class later on that would inherit BookRepository.cs.

Reply:
Okay, I leave it up to you :)